otter/convergence/gathering.py

```diff
@@ -84,21 +83,27 @@ def group_id(s):
     return get_all_server_details(changes_since).on(servers_apply)
 
 
-def merge_servers(first, second):
+def updated_deleted_servers(old, new):
     """
-    Return servers got by merging the two sets of servers where second
-    takes precedence over first
+    Return servers with servers not found in new list marked as deleted
```

Comment:
> The name of this method is a bit weird, I guess maybe it was meant to be update_deleted_servers (i.e. update instead of updated?)

How about `mark_deleted_servers`?

> Maybe "Given dictionaries containing old and new servers, return a list of all servers, with the deleted ones annotated with a status of DELETED."

This is great docstring. Thanks!